harness/determined/experimental/client.py

```diff
@@ -447,3 +447,86 @@ def stream_trials_validation_metrics(trial_ids: List[int]) -> Iterable[Validatio
 def _get_singleton_session() -> Session:
     assert _determined is not None
     return _determined._session
+
+
+@_require_singleton
+def bind_rps_to_workspaces(
+    resource_pool_names: List[str],
```

Comment:
same comment as above, I think we want to only bind to one resource pool at a time